package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
```

Comment:
please add the packages for react-native in client/. Remove package.json from the root folder of the project.